EIPS/eip-3326.md

```diff
@@ -0,0 +1,130 @@
+---
+eip: 3326
+title: Wallet Switch Ethereum Chain RPC Method (`wallet_switchEthereumChain`)
+author: Erik Marks (@rekmarks)
+discussions-to: https://ethereum-magicians.org/t/eip-3326-wallet-switchethereumchain
+status: Draft
+type: Standards Track
+category: Interface
+created: 2021-03-04
+requires: 155, 695
+---
+
+## Simple Summary
+
+An RPC method for switching the wallet's active Ethereum chain.
+
+## Abstract
+
+The `wallet_switchEthereumChain` RPC method allows Ethereum applications ("dapps") to request that the wallet switches its active Ethereum chain, if the wallet has a concept thereof.
+The caller must specify a chain ID.
+The wallet application may arbitrarily refuse or accept the request.
+`null` is returned if the active chain was switched, and an error otherwise.
+
+Important cautions for implementers of this method are included in the [Security Considerations](#security-considerations) section.
```

Comment:
I think this line is unnecessary, security considerations should already be an important section for implementers to read :)

Reply:
EIPS/eip-3326.md